spec/filters/drop_spec.rb

```diff
@@ -16,16 +16,30 @@
       subject.filter(event)
       expect(event).to be_cancelled
     end
-
+  
     context "when using percentage" do
-      let(:config) { { "percentage" => 0 }}
+      let(:config) { { "percentage" => 100 }}
```

Comment:
Add `, "add_field" => { "field1" => "pass" }` here and verify with `expect(event.get("field1")).to be_nil`